storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -136,12 +121,67 @@ The backup is now prepared and can be restored with the :option:`--copy-back` op
 
    $ xtrabackup --copy-back --target-dir=/data/backup --datadir=/data/mysql
 
+
+Using the ``keyring_file`` component
+-------------------------------------
+
+A component is not loaded with the ``--early_plugin_load`` option. The server uses a manifest to load the component and the component has its own configuration file. See `component installation <https://dev.mysql.com/doc/refman/8.0/en/keyring-component-installation.html>`__ for more information.
+
+The following is an example of a manifest and a configuration file:
+
+.. sourcecode::
+
+   ./bin/mysqld.my: { "components": "file://component_keyring_file" }
```

Comment:
Note: This is valid for global manifest and global configuration, when mysql tarball is used. Probably format it better, refer https://dev.mysql.com/doc/refman/8.0/en/keyring-file-component.html

Reply:
fixed and added.